### DISCUSSION TITLE
What's archetype.foundation?

### DISCUSSION THREAD

**Discussion by metasean** [retreon/retreon#249] [Q&A] [answered]
Inquiring minds want to know, what is achetype.foundation?

**Comment by PsychoLlama** [comment, marked as answer]
Woot! I get to try the discussions feature! 🙌

Good question. Most of my projects are barely past proof of concept ([arc](https://github.com/PsychoLlama/arc), [piece-table](https://github.com/PsychoLlama/piece-table.rs), [navitron.vim](https://github.com/PsychoLlama/navitron.vim)) or maintained _juuuust enough_ to stay useful (like [bin-json](https://github.com/PsychoLlama/bin-json), [hue-connect](https://github.com/PsychoLlama/hue-connect), [freeform-input](https://github.com/PsychoLlama/freeform-input)). For the most part, once they're done, they're done. I barely maintain them.

But there are a few projects useful enough that they warrant continued maintenance: retreon, [freighter](https://github.com/PsychoLlama/freighter), [dispute](https://github.com/PsychoLlama/dispute/) - although the last two haven't gotten much love recently.

The Archetype Foundation is a conceptual bucket where I can put projects that are "finished", general purpose, and maintained. The name came from a conversation with my brother where I described my sabbatical as "self-employed at a non-profit". If it has practical benefits, I might register it officially. I dunno, haven't thought that far. I just have the domain 🙂
